.github/workflows/flake8.yml

```diff
@@ -0,0 +1,39 @@
+name: "flake8"
```

Comment:
Created a separate workflow as the pep8 one was manually set to not run anymore.